app/controllers/users_controller.rb

```diff
@@ -21,29 +23,15 @@ def update
     end
   end
 
-  def register
-    if request.put?
-      handle_with(UsersRegister,
-                  contracts_required: !contracts_not_required,
-                  success: lambda {
-                    redirect_back key: :registration_return_to
-                  },
-                  failure: lambda {
-                    errors = @handler_result.errors.any?
-                    render :register, status: errors ? 400 : 200
-                  })
-    else
-      store_fallback key: :registration_return_to
-    end
-  end
-
   def ask_for_email
+    # FWIW, when this code is this resurrected, the registration_redirect_url
+    # may not be what we need.
```

Comment:
When we ask for email in the future, I don't think it'll be part of the registration process -- it'll just be us seeing that someone who logged in does not have a registered email address and we'll want to solicit one.  This note here is just saying that `registration_redirect_url` might be misleading for future developers of that use case.